packages/pcd/gpc-pcd/src/GPCPCDPackage.ts

```diff
@@ -251,23 +271,42 @@ export function getProveDisplayOptions(): ProveDisplayOptions<GPCPCDArgs> {
         argumentType: ArgumentTypeName.String,
         defaultVisible: true
       },
-      pod: {
-        argumentType: ArgumentTypeName.PCD,
+      pods: {
+        argumentType: ArgumentTypeName.Record,
         description: "Generate a proof for the selected POD object",
-        validate(value, _params): boolean {
-          if (value.type !== PODPCDTypeName) {
-            return false;
-          }
+        validate:
+          (podName) =>
+          (podPCD, params): boolean => {
+            if (podPCD.type !== PODPCDTypeName) {
+              return false;
+            }
+
+            if (params?.proofConfig !== undefined) {
+              const proofConfig = deserializeGPCProofConfig(params.proofConfig);
```

Comment:
This can throw.  I think it may be best not to throw from the validator, but simply return false, with a customized notFoundMessage